src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import Hero from "./components/hero/Hero";
+
+export const App: React.FC = () => {
```

Comment:
While I also use it every once in a while, in general strong-typing your component with `React.FC` is in general [discouraged](https://github.com/facebook/create-react-app/pull/8177)